.github/workflows/release-notes-monitor.yml

```diff
@@ -17,46 +23,52 @@ jobs:
         fetch-depth: 0
         ref: main
 
-    - name: Check for a Release Notes tag for this commit
-      uses: nick-fields/retry@3e91a01664abd3c5cd539100d10d33b9c5b68482 # pin@v2
-      with:
-        max_attempts: 2
-        timeout_seconds: 30
-        command: |
-          echo "new_commit_hash=$(git rev-parse HEAD)" >> $GITHUB_ENV
-          echo "existing_tag=$(git tag --points-at ${{ env.commit_hash }} | grep -E 'release_notes_v(.*)' | head -n 1)" >> $GITHUB_ENV
-          echo "sui_version=$(cat Cargo.toml | grep "^version =" | tr -d '"' | awk '{ print $3 }')" >> $GITHUB_ENV
-          [[ -n ${{ env.existing_tag }} ]] && echo "old_commit_hash=$(git show-ref -s ${{ env.existing_tag }})" >> $GITHUB_ENV
-
     - name: Get list of PRs
       id: get-pr-list
-      if: ${{ env.old_commit_hash != '' }}
       working-directory: ./
       run: |
-        export list_of_prs=$(git log --grep "\[x\]" --pretty=oneline --abbrev-commit d8de3b747ba275542825d3d7dfe78067e3c88df...8c0c8780447e519db3eb3644492343d0bbf179ef -- crates dashboards doc docker external-crates kiosk narwhal nre sui-execution | grep -o '#[0-9]\+' | grep -o '[0-9]\+')
+        export new_commit_hash=$(git rev-parse HEAD)
+        export existing_commit_tag=$(git tag --points-at ${new_commit_hash} | grep -E 'sui_v(.*)_rel_notes' | head -n 1)
+
+        export previous_tag=$(git tag | grep -E 'sui_v1(.*)_rel_notes' | sort -r | head -1)
+        export previous_commit_hash=$(git rev-list -n 1 ${previous_tag})
+        
+        export list_of_prs=$(git log --grep "\[x\]" --pretty=oneline --abbrev-commit ${previous_commit_hash}...${new_commit_hash} -- crates dashboards doc docker external-crates kiosk narwhal nre sui-execution | grep -o '#[0-9]\+' | grep -o '[0-9]\+' | jq -R -s -c 'split("\n")[:-1]')
+
         echo "::set-output name=matrix::${list_of_prs}"
-  
+        echo "::set-output name=new_commit_hash::${new_commit_hash}"
```

Comment:
set-output and save-state are deprecated.  though from the blog they've delayed it a bit.